releng/test_config.yaml

```diff
@@ -35,35 +35,34 @@ jobs:
 
   # release branch jobs - cos only (cvm is deprecated at 1.8)
   # beta release (inactive)
-  # TODO(1.28): Uncomment this when adding jobs for release-1.28 branch.
-  # ci-kubernetes-e2e-gce-cos-k8sbeta-reboot:
-  #   interval: 1h
-  #   sigOwners: [sig-cloud-provider-gcp]
-  #   releaseBlocking: true
-  # ci-kubernetes-e2e-gce-cos-k8sbeta-ingress:
-  #   interval: 1h
-  #   sigOwners: [sig-cloud-provider-gcp]
-  #   releaseBlocking: true
-  # ci-kubernetes-e2e-gce-cos-k8sbeta-default:
-  #   args:
-  #   - --env=ENABLE_CACHE_MUTATION_DETECTOR=true
-  #   interval: 1h
-  #   sigOwners: [sig-cloud-provider-gcp]
-  #   releaseBlocking: true
-  # ci-kubernetes-e2e-gce-cos-k8sbeta-serial:
-  #   interval: 1h
-  #   sigOwners: [sig-cloud-provider-gcp]
-  #   releaseInforming: true
-  #   testgridNumFailuresToAlert: 6
-  # ci-kubernetes-e2e-gce-cos-k8sbeta-slow:
-  #   interval: 1h
-  #   sigOwners: [sig-cloud-provider-gcp]
-  #   releaseInforming: true
-  #   testgridNumFailuresToAlert: 6
-  # ci-kubernetes-e2e-gce-cos-k8sbeta-alphafeatures:
-  #   interval: 1h
-  #   sigOwners: [sig-cloud-provider-gcp]
-  #   releaseBlocking: true
+  ci-kubernetes-e2e-gce-cos-k8sbeta-reboot:
+    interval: 1h
+    sigOwners: [sig-cloud-provider-gcp]
+    releaseBlocking: true
+  ci-kubernetes-e2e-gce-cos-k8sbeta-ingress:
+    interval: 1h
+    sigOwners: [sig-cloud-provider-gcp]
+    releaseBlocking: true
+  ci-kubernetes-e2e-gce-cos-k8sbeta-default:
+    args:
+    - --env=ENABLE_CACHE_MUTATION_DETECTOR=true
```

Comment:
Do we have some idea if this arg is still relevant? It doesn't appear in stable1 and it hasn't been added recently, so maybe we should somehow revisit this.

Reply:
Still used by sig cloud provider jobs: https://github.com/kubernetes/test-infra/blob/b11b77d3d71e77c88a172a5171c6c06fc88db9f6/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L48-L49. Not sure if we want the same signal for this job.

Reply:
Okay, let's leave it as it is